.github/workflows/web-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Test web code
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/web-test.yml"
+      - "web-admin/**"
+      - "web-cloud/**"
```

Comment:
This should be `web-common`

Reply:
oh yeah, for some reason in my local there is web-cloud too